rubocop.yml

```diff
@@ -0,0 +1,15 @@
+Layout:
+  Enabled: false
+
+Style/MultilineIfModifier: # clashes with rubyModifier
```

Comment:
Also for this one with the default config you don't need to turn this off. Since you would have to change the config anyway if you want to have non-default rubocop config I'm tempted to just keep this out.